#!/usr/bin/env python
PACKAGE = "vision_module"

from dynamic_reconfigure.parameter_generator_catkin import *
 
gen = ParameterGenerator()
 
gen.add("MAX_DIST", double_t, 0, "MAX DISTANCE", 8.0, 0.4, 8.0)
gen.add("MIN_DIST", double_t, 0, "MIN DISTANCE", 0.5, 0.4, 8.0)
gen.add("ERROR_RANGE", double_t, 0, "ERROR RANGE", 0.012, 0.0, 0.05)
gen.add("MIN_SEGMENT_SIZE", int_t, 0, "MIN SEGMENT SIZE", 20000, 4000, 80000)
gen.add("SEARCH_ITERATION", int_t, 0, "SEARCH ITERATION", 400, 100, 800)
gen.add("MAX_GAP", double_t, 0, "MAX GAP", 0.05, 0.01, 0.1)
gen.add("DISPLAY", bool_t, 0, "DISPLAY RESULT", False)
gen.add("SAVE_IMAGE", bool_t, 0, "SAVE IMAGE", False)

exit(gen.generate(PACKAGE, "plane_detection", "plane_detection"))
